tests/pos/suspend-strawman-2/monadic-reflect.scala

```diff
@@ -15,9 +15,8 @@ trait Monad[F[_]]:
 
 end Monad
 
-trait CanReflect[M[_], R] {
-  def reflect(mr: M[R])(using Monad[M]): R
-}
+trait CanReflect[M[_]: Monad, R]:
```

Comment:
Is the context bound here really necessary? In our [monadic reflection](https://github.com/lampepfl/monadic-reflection/blob/main/core/src/main/scala/monadic/CanReflect.scala) implementation, we do not specify the bound here. The moment we need the fact that `M` is a monad is only at `reify`.

Reply:
Indeed, the bound is not necessary.